src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -100,6 +100,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
  * Default configuration, to allow one single parameterless annotation on the same line:
```

Comment:
make it `To configure the default check to ....` and 
similar to https://github.com/checkstyle/checkstyle/pull/8641#discussion_r468624772

Reply:
Done